tests/transforms/test_mitigate.py

```diff
@@ -363,6 +363,24 @@ def qfunc(theta):
 class TestDifferentiableZNE:
     """Testing differentiable ZNE"""
 
+    @pytest.mark.parametrize("lambda_", [1.0, 1.5, 1.7, 2.0, 2.5])
+    @pytest.mark.parametrize("num_ops", [4, 8, 9, 12, 16])
+    def test_correct_number_of_operators(self, lambda_, num_ops):
+        """Test the output corresponds to the right number of operators according to https://arxiv.org/abs/2005.10921"""
```

Comment:
Thanks for giving the reference, very useful :)
```suggestion
        """Test the output corresponds to the right number of operators according to Sec. II A 1) in  https://arxiv.org/abs/2005.10921"""
```